### ISSUE TITLE
Add eslint

### ISSUE THREAD

**Issue by diguifi** [tiny-devs/tiny-dungeon-online#7]
Add eslint to project

**Comment by JoelLau** [comment]
hasn't [tslint been depracated](https://blog.palantir.com/tslint-in-2019-1a144c2317a9)?

**Comment by diguifi** [comment]
@JoelLau You're absolutely correct! This old issue is mostly to make us aware that the entire codebase needs some basic linting. I'll fix the title!
